app/src/main/java/mod/tsd/editor/library/ExtDepedency.java

```diff
@@ -0,0 +1,91 @@
+package mod.tsd.editor.library;
+
+import a.a.a.jq;
+import java.util.Optional;
+import java.util.ArrayList;
+import java.util.List;
+import mod.agus.jcoderz.handle.component.ConstVarComponent;
+import mod.agus.jcoderz.editor.library.ExtLibSelected;
+import mod.jbk.build.BuiltInLibraries;
+import mod.jbk.editor.manage.library.ExcludeBuiltInLibrariesActivity;
+
+public class ExtDepedency {
+	public static String dependency;
+	public static ConstVarComponent constVarComponent;
+	public static List<BuiltInLibraries.BuiltInLibrary> excludedLibraries;
```

Comment:
none of these should be fields - they're only used in `addExtDepedency(jq, String)`